### ISSUE TITLE
Design for annotations show page

### ISSUE THREAD

**Issue by mejackreed** [sul-dlss/exhibits#727]

**Comment by ggeisler** [comment]
<img width="1066" alt="annotation-show-page" src="https://user-images.githubusercontent.com/101482/31518497-4c37694e-af54-11e7-88e8-ee928d862eaf.png">

**Comment by blalbrit** [comment]
@ggeisler - how did you anticipate that we would get the "section" information? If I understand what that info is, I would specify this as a nice-to-have (not must or should) based on how that information might be retrieved.

**Comment by ggeisler** [comment]
@blalbrit I think I was assuming that since we know the page of a manuscript the annotation is associated with, and we know (I think?) the named section that a given page falls within, we should be able to know the named section associated with the annotation.

However, I have no idea how easy it is to apply that logic to indexing the section field for the annotation. I have no problem omitting it if it seems difficult to index. If it's straightforward to index, I'd suggest it could be potentially useful to the user and we might consider including it.But  I'd certainly defer to you on the indexing effort/user value tradeoff if it isn't very easy to index.

**Comment by drh-stanford** [comment]
I think the section/page are the only issues with the design, and @blalbrit has marked them nice-to-haves. I'm going to close this design ticket and further work will be in #749. Please reopen as needed.

**Comment by drh-stanford** [comment]
Based on a 10/26 discussion, we're interested in making the AnnotationList (i.e., Canvas, manuscript page) the first-class object. So, the action item here is to revisit the design with that idea.

**Comment by ggeisler** [comment]
Below is a simple idea for the Page Details (formerly Annotation) show page. The user gets to this page from a Page Details search result item (Option D in https://github.com/sul-dlss/exhibits/issues/726#issuecomment-342293622).

----
<img width="1238" alt="page-details-show-page" src="https://user-images.githubusercontent.com/101482/32506496-f76f05b8-c3a1-11e7-88fc-d30cc48bf1f8.png">

----

A potential enhancement would be to link each annotation in the list to its canvas location (e.g., in the version of the annotation list below clicking the view icon would highlight the annotation box of the selected annotation). From a previous conversation with @aeschylus, though, I gather a future enhancement to Mirador is to show the annotation list in the Mirador sidebar and directly link that list with the annotation boxes on the canvas. If my understanding of that is correct, it might make more sense to wait on that enhancement and stick with the simpler annotation list as shown in the first mockup above.

----

<img width="734" alt="linked-annotations-list" src="https://user-images.githubusercontent.com/101482/32506922-203eb956-c3a3-11e7-92e0-9b84c06ee2ce.png">
